backend/app/controllers/spree/admin/orders_controller.rb

```diff
@@ -61,7 +61,7 @@ def index
 
       def new
         user = Spree.user_class.find_by(id: params[:user_id]) if params[:user_id]
-        @order = Spree::Core::Importer::Order.import(user, order_params)
+        @order = Spree::Core::Importer::Order.import(user, order_params(user))
```

Comment:
I'd go with something like this:

```ruby
def new
  user = Spree.user_class.find_by(id: params[:user_id]) if params[:user_id]
  order_importer_params = order_params
  order_importer_params.merge({
    ship_address: user.ship_address,
    bill_address: user.bill_address
  }) if user

  @order = Spree::Core::Importer::Order.import(user, order_importer_params)
  redirect_to cart_admin_order_url(@order)
end
```

Anyway, probably with another PR, it could make sense to deprecate passing the `user` argument as a separate one, while we could just add it the `order_importer_params` hash. I'm not sure why it's this way and may be worth exploring.

Reply:
Interesting approach. I think I agree with it since passing `user` to `order_params` seems odd.